runtime/testnet/src/lib.rs

```diff
@@ -191,18 +214,13 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	impl_name: Cow::Borrowed("pop"),
 	authoring_version: 1,
 	#[allow(clippy::zero_prefixed_literal)]
-	spec_version: 00_04_02,
+	spec_version: 00_04_03,
```

Comment:
nit: seems like this should be a major bump as we are introducing new features: `00_05_00`. 

Non-blocking though.